examples/kafka-hub/consolidator/docker/alpine/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# syntax = docker/dockerfile:1.0-experimental
```

Comment:
What is the usage of this line?

Reply:
this is used to override the default frontends. I removed it as it is not needed.